devel/tmt/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                tmt
+version             1.34.0
+revision            0
+
+categories          devel
+supported_archs     noarch
+license             MIT
+maintainers         {sub-pop.net:link @subpop} \
+                    openmaintainer
+
+description         Test Management Tool
+long_description    The tmt tool provides a user-friendly way to work with \
+                    tests. You can comfortably create new tests, safely and \
+                    easily run tests across different environments, review \
+                    test results, debug test code and enable tests in the CI \
+                    using a consistent and concise config. \
+                    The python module and command-line tool implement the \
+                    Metadata Specification which allows storing all needed \
+                    test execution data directly within a git repository. \
+                    Together with the possibility to reference remote \
+                    repositories, it makes it easy to share test coverage \
+                    across projects and distros. \
+                    The Flexible Metadata Format fmf is used to store data in \
+                    both human and machine readable way close to the source \
+                    code. Thanks to inheritance and elasticity metadata are \
+                    organized in the structure efficiently, preventing \
+                    unnecessary duplication.
+
+homepage            https://tmt.readthedocs.io
+
+checksums           rmd160  08aac1c5b146b5b8d0fa2dfd638ac319e68da6f9 \
+                    sha256  e3b8b0d0af1dd5d56522c9ebc8c6ccb4bfb04e6cf3e54f31402cf8dbc1717cc4 \
+                    size    605318
+
+python.default_version 312
+python.pep517_backend  hatchling
```

Comment:
the backend is called "hatch" - see the "python" PortGroup. That should also automatically add the "hatchling" build dependency.

Reply:
Ah thank you!